boards/arm64/xenvm/xenvm-gicv2-qemu_defconfig

```diff
@@ -0,0 +1,30 @@
+CONFIG_SOC_XENVM=y
+CONFIG_BOARD_XENVM=y
+CONFIG_XEN_INITIAL_DOMAIN=y
+
+# Enable UART driver
+CONFIG_SERIAL=y
+
+CONFIG_XIP=n
+CONFIG_FLASH_BASE_ADDRESS=0x0
+CONFIG_FLASH_SIZE=0
+CONFIG_AARCH64_IMAGE_HEADER=y
+CONFIG_MAX_XLAT_TABLES=10
+CONFIG_HEAP_MEM_POOL_SIZE=262144
+
+# Enable console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# Enable PV hypervisor console
+CONFIG_UART_CONSOLEIO_XEN_HVC=y
```

Comment:
This code is not buildable.` CONFIG_UART_CONSOLEIO_XEN_HVC `-> `CONFIG_UART_XEN_HVC_CONSOLEIO`